include/tester/ssp/Softphone.hpp

```diff
@@ -66,7 +65,8 @@ class Softphone
 
 private:
     static constexpr auto SIP = "sip:";
-
+    std::function<void(pj::CallInfo ci, const pj::OnCallStateParam &prm)> boundOnCallState =
+        std::bind(&Softphone::onCallState, this, std::placeholders::_1, std::placeholders::_2);
```

Comment:
I see no reason for having this variable. Even though you're only creating one instance of the function and copying it to the SSPCall instances, I'd rather you use std::bind for when necessary.

Reply:
done